### PR TITLE
add `format_number_no_round` function

### PR DESCRIPTION
**Pull request by joshuadavidthomas**

### CODE REVIEW

File: tests/test_decimal.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    assert isinstance(result, Decimal)
+    assert result == expected
+
+
+@pytest.mark.parametrize(
+    "decimal_places, expected",
+    [
+        (0, "123.456789"),
+        (2, "123.456789"),
```
Comment (jefftriplett) [inline tests/test_decimal.py:101]: This throws me.

Comment (joshuadavidthomas) [reply tests/test_decimal.py:101]: Ah, yeah.. Not in love with the name of the function, which may go a little ways to explain what the purpose is (beyond, you know, some actual docs....).

For display in a template, I needed a quick way to format a dollar amount to two decimal places without rounding.

E.g.

123.4 -> 123.40
123.45 -> 123.45
123.456 -> 123.456
123.4567 -> 123.4567
123.45670 -> 123.4567

and so on.

Comment (joshuadavidthomas) [reply tests/test_decimal.py:101]: Or three decimal places, which is why I added the `decimal_places` arg. We have some tracts of land that have fractional acres, but the majority are whole numbers. So I wanted to just show 123 if the acres were 123.000, but 123.456 if the acres were 123.456.

I don't think I've actually pushed that local change actually.

Comment (joshuadavidthomas) [reply tests/test_decimal.py:101]: I need to clean it up a bit to clarify things, because even I'm confusing myself thinking through it right now.. 😅

**Review by jefftriplett — APPROVED** [review]

**Pull request merged**